src/ResourceManager/AzureBatch/Commands.Batch/Microsoft.Azure.Commands.Batch.format.ps1xml

```diff
@@ -6,39 +6,26 @@
       <ViewSelectedBy>
         <TypeName>Microsoft.Azure.Commands.Batch.BatchAccountContext</TypeName>
       </ViewSelectedBy>
-      <ListControl>
-        <ListEntries>
-          <ListEntry>
-            <ListItems>
-              <ListItem>
-                <PropertyName>AccountName</PropertyName>
-              </ListItem>
-              <ListItem>
-                <PropertyName>Location</PropertyName>
-              </ListItem>
-              <ListItem>
-                <PropertyName>ResourceGroupName</PropertyName>
-              </ListItem>
-              <ListItem>
-                <PropertyName>CoreQuota</PropertyName>
-              </ListItem>
-              <ListItem>
-                <PropertyName>PoolQuota</PropertyName>
-              </ListItem>
-              <ListItem>
-                <PropertyName>ActiveJobAndJobScheduleQuota</PropertyName>
-              </ListItem>
-              <ListItem>
-                <Label>Tags</Label>
-                <PropertyName>TagsTable</PropertyName>
-              </ListItem>
-              <ListItem>
-                <PropertyName>TaskTenantUrl</PropertyName>
-              </ListItem>
-            </ListItems>
-          </ListEntry>
-        </ListEntries>
-      </ListControl>
+      <CustomControl>
+        <CustomEntries>
+          <CustomEntry>
+            <CustomItem>
+              <Frame>
+                <CustomItem>
+                  <ExpressionBinding>
+                    <!-- Omit the keys from display by default. -->
+                    <!-- Also omit Tags, since there's a separate TagsTable property specifically for display purposes. -->
+                    <ScriptBlock>
+                      $str = $_ | Select-Object -Property * -ExcludeProperty PrimaryAccessKey,SecondaryAccessKey,Tags | Format-List -Property * | Out-String
```

Comment:
It would be nice to show an excepted value for the string properties, so you know they are there (like a generic 'xxxxxx' for each one)